openseadragon.module

```diff
@@ -148,6 +157,14 @@ function template_preprocess_openseadragon_iiif_manifest_block(&$variables) {
         'id' => $openseadragon_viewer_id,
         'prefixUrl' => 'https://cdnjs.cloudflare.com/ajax/libs/openseadragon/2.4.2/images/',
         'tileSources' => $tile_sources,
+
+        // For dsu-utsc.
+        'loadTilesWithAjax' => TRUE,
+        'ajaxWithCredentials' => TRUE,
+        'ajaxHeaders' => [
+          "Authorization" => "Bearer " . $access_token,
+          'token' => $access_token,
+        ],
```

Comment:
Much the same as the [previous PR](https://github.com/Islandora/openseadragon/pull/40#discussion_r770804718), these tokens are generally expected to be valid for a limited length of time, so we would have to propagate cache headers preventing them from being used beyond their lifetime to prevent serving up responses with invalid tokens.